tests/basic/storage_pool_test.go

```diff
@@ -2374,6 +2374,24 @@ var _ = Describe("{VolUpdateResizeDisk}", func() {
 		})
 		err = ValidateReplFactorUpdate(volSelected, newRep+1)
 		log.FailOnError(err, "error validating repl factor for vol [%s]", volSelected.Name)
+
+		stepLog = "Initiate pool expansion using resize-disk after rsync is successfull"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			poolToBeResized, err = GetStoragePoolByUUID(selectedPool.Uuid)
+
+			drvSize, err := getPoolDiskSize(poolToBeResized)
+			log.FailOnError(err, "error getting drive size for pool [%s]", poolToBeResized.Uuid)
+			expectedSize := (poolToBeResized.TotalSize / units.GiB) + drvSize
+
+			log.InfoD("Current Size of the pool %s is %d", selectedPool.Uuid, poolToBeResized.TotalSize/units.GiB)
+			err = Inst().V.ExpandPool(selectedPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize)
```

Comment:
should it be ADD_DISK instead of RESIZE_DISK ?

in summary its mentioned as add_disk 


//2) Select a pool with iops happening.
	//3) Expand pool by add-disk
	//4) Expand pool again by add-disk with different size multiple.
	//4) Expand pool again by radd-disk with different size multiple.
	//4) Expand pool again by add-disk with different size multiple.